core/tls_log_filter_test.go

```diff
@@ -1,46 +1,32 @@
 package core
 
-import (
-	"fmt"
-	"log"
-	"regexp"
-	"testing"
+// func TestTLSLogFilter_Write(t *testing.T) {
```

Comment:
Is this code not required any more or is it still a work in progress? If it's not required then it should be removed.